pkg/apis/psmdb/v1/perconaservermongodbrestore_types.go

```diff
@@ -18,6 +18,26 @@ type PerconaServerMongoDBRestoreSpec struct {
 	BackupSource *PerconaServerMongoDBBackupStatus `json:"backupSource,omitempty"`
 	StorageName  string                            `json:"storageName,omitempty"`
 	PITR         *PITRestoreSpec                   `json:"pitr,omitempty"`
+	Selective    *SelectiveRestoreOpts             `json:"selective,omitempty"`
```

Comment:
Selective restore is for logical backups only. We need to have validation for it. If we have physical backup we should not run restore at all.

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1650/commits/46324fa8ff3f08b6559e672a09ef8f8f00940486